tests/integration-test.js

```diff
@@ -67,11 +66,11 @@ describe('component render test', function () {
       // 2
       // 6
       calls.render[props.cursor.toJS().id]++;
-      return d.li({}, '');
+      return React.createElement('li', {}, '');
```

Comment:
All of the changes to use `React.createElement` were to avoid warnings, based on the React team's [recommendations](https://facebook.github.io/react/docs/react-api.html#createelement).